benchcab/config.py

```diff
@@ -67,6 +67,28 @@ def validate_config(config: dict) -> bool:
     raise ConfigValidationException(v)
 
 
+def read_optional_data(config: dict):
+    if "realisations" in config:
+        for r in config["realisations"]:
+            r["name"] = r.get("name")
+
+    config["science_configurations"] = config.get(
+        "science_configurations", internal.DEFAULT_SCIENCE_CONFIGURATIONS
+    )
+
+    config["fluxsite"] = config.get("fluxsite", {})
+
+    config["fluxsite"]["multiprocess"] = config["fluxsite"].get(
+        "multiprocess", internal.FLUXSITE_DEFAULT_MULTIPROCESS
+    )
+    config["fluxsite"]["experiment"] = config["fluxsite"].get(
+        "experiment", internal.FLUXSITE_DEFAULT_EXPERIMENT
+    )
+    config["fluxsite"]["pbs"] = internal.FLUXSITE_DEFAULT_PBS | config["fluxsite"].get(
+        "pbs", {}
+    )
```

Comment:
Here we need to set each key of `pbs` dictionary to the default value if not set. See comment on pbs.py.

Reply:
I believe that's the behavior here (since python 3.9)?

Reply:
Yes, no worries.